src/features/fireWeather/components/graphs/WxDataGraphToggles.tsx

```diff
@@ -4,178 +4,165 @@ import { makeStyles } from '@material-ui/core/styles'
+            name="showHighResModels"
+            data-testid="wx-graph-high-res-model-toggle"
+            checked={toggleValues.showHighResModels}
+            disabled={noHighResModels}
             size="small"
-            onChange={(_, checked) => {
-              setShowPastForecasts(checked)
+            onChange={(e, checked) => {
+              setToggleValues(e.target.name as 'showHighResModels', checked)
             }}
           />
         }
         label={
-          <Typography className={classes.label} variant="body2">
-            Historic Noon Forecasts
+          <Typography className={classes.switchLabel} variant="body2">
+            HRDPS
           </Typography>
         }
       />
+
       <FormControlLabel
-        className={classes.formControlLabel}
+        className={classes.switchControl}
         control={
           <Switch
             name="showModels"
             data-testid="wx-graph-model-toggle"
-            checked={showModels}
+            checked={toggleValues.showModels}
             disabled={noModels}
             size="small"
-            onChange={(_, checked) => {
-              setShowModels(checked)
-            }}
-          />
-        }
-        label={
-          <Typography className={classes.label} variant="body2">
-            Models
-          </Typography>
-        }
-      />
-      <FormControlLabel
-        className={classes.formControlLabel}
-        control={
-          <Switch
-            name="showForecasts"
-            data-testid="wx-graph-forecast-toggle"
-            checked={showForecasts}
-            disabled={noForecasts}
-            size="small"
-            onChange={(_, checked) => {
-              setShowForecasts(checked)
-            }}
+            onChange={handleSwitch}
           />
         }
         label={
-          <Typography className={classes.label} variant="body2">
-            Noon Forecasts
+          <Typography className={classes.switchLabel} variant="body2">
+            GDPS
           </Typography>
         }
       />
+
       <FormControlLabel
-        className={classes.formControlLabel}
+        className={classes.switchControl}
         control={
           <Switch
-            name="showBiasAdjustedPredictions"
+            name="showBiasAdjustedModels"
             data-testid="wx-graph-bias-toggle"
-            checked={showBiasAdjustedPredictions}
-            disabled={noBiasAdjustedPredictions}
+            checked={toggleValues.showBiasAdjustedModels}
+            disabled={noBiasAdjustedModels}
             size="small"
-            onChange={(_, checked) => {
-              setShowBiasAdjustedPredictions(checked)
-            }}
+            onChange={handleSwitch}
           />
         }
         label={
-          <Typography className={classes.label} variant="body2">
-            Bias Adjusted Models
+          <Typography className={classes.switchLabel} variant="body2">
+            Bias Adjusted GDPS
```

Comment:
how much extra work would it be to do Bias Adjusted HRDPS? Not suggesting we do it as part of this PR, we should create a new ticket for it, but I'm just curious about how much work that new ticket would be.

Reply:
It would be only a small amount of work since the data is available on the FE already. We will have to create a toggle and display it in the graph. (It can vary based on how we are gonna display GDPS and HRDPS in the near future)